src/doc/guide.md

```diff
@@ -570,11 +562,10 @@ if x == 5i {
 }
 ```
 
-If we changed the value of `x` to something else, this line would not print.
-More specifically, if the expression after the `if` evaluates to `true`, then
-the block is executed. If it's `false`, then it is not.
+As we might expect, the block is executed when the `if` guard expression
+evaluates to `true`. If it's `false`, the block is skipped.
 
-If you want something to happen in the `false` case, use an `else`:
+Here we can see an `else` block:
```

Comment:
:+1:  simply introducing the example with "adding an `else` works exactly like we would expect" should be sufficient. I'm not sure programmers need `else` blocks explained to them. Rust isn't that different here.